### PR TITLE
Various fixes, mapping file seems not to be applied

### PR DESCRIPTION
**Pull request by RichiH**
Trying to fix a few affiliations.

NB: The mapping file seems to not be applied correctly.

### CODE REVIEW

**Comment by gouthamve** [comment]
cc @superq

File: src/maintainers.csv [hunk 1]
```diff
@@ -18,8 +18,8 @@ Callum Styan,Grafana Labs,cstyan
 Chris Marchbanks,Splunk,csmarchbanks
 Fabian Reinartz,Google,fabxc
 Frederic Branczyk,Red Hat,brancz
-Ganesh Vernekar,Grafana,codesome
-Goutham Veeramachaneni,Grafana,gouthamve
+Ganesh Vernekar,Grafana Labs,codesome
+Goutham Veeramachaneni,Grafana Labs,gouthamve
```
Comment (gotjosh) [inline src/maintainers.csv:22]:
```suggestion
Goutham Veeramachaneni,Grafana Labs,gouthamve
Josh Abreu, Grafana Labs,gotjosh
```

**Review by SuperQ — APPROVED** [review]

**Comment by lukaszgryglicki** [comment]
Actually current process is not using `maintainers.csv` file, all details about adding affiliations are [here](https://github.com/cncf/gitdm#addingupdating-affiliation), so I'm not sure what do you refer to when saysing that `NB: The mapping file seems to not be applied correctly.`.

**Comment by lukaszgryglicki** [comment]
I will `try` to do corresponding updates in the `github_users.json` file and `developers_affiliationsN.txt` files.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm but I will do the changes manually in post-merge.

**Comment by lukaszgryglicki** [comment]
Merged and checked affiliations files, they were already OK, you can just check the following files to make sure all is OK:
- `src/cncf/config/email-map`.
- `src/github_users.json`.

**Pull request merged**